### DISCUSSION TITLE
Misdeleted the heartbeat cronjob

### DISCUSSION THREAD

**Discussion by ywtung** [linkerd/linkerd2#8729] [Help]
I was trying to disable the heartbeat by editing the `ConfigMap` of Linkerd but after I finished it the heartbeat seems not to be disabled, after that I misdeleted the heartbeat cronjob, is there any way to recover the cronjob? Or is it effected to the whole Linkerd system?

```
Environment: EKS
Version: Stable 2.11.1
Kubernetes Version: 1.22.4
```
I would greatly appreciate it if who knows the answer and helps.

**Comment by alpeb** [comment]
You can just perform a linkerd upgrade, even if it's to the same version you already have installed. Either through the CLI or Helm. That should reestablish the erased CronJob.
